cli/smartcontract/smart_contract.go

```diff
@@ -100,22 +112,14 @@ func NewCommands() []*cli.Command {
 	}
 	invokeFunctionFlags = append(invokeFunctionFlags, options.Wallet...)
 	invokeFunctionFlags = append(invokeFunctionFlags, options.RPC...)
-	deployFlags := append(invokeFunctionFlags, []cli.Flag{
-		&cli.StringFlag{
-			Name:     "in",
-			Aliases:  []string{"i"},
-			Required: true,
-			Usage:    "Input file for the smart contract (*.nef)",
-			Action:   cmdargs.EnsureNotEmpty("in"),
-		},
-		&cli.StringFlag{
-			Name:     "manifest",
-			Aliases:  []string{"m"},
-			Required: true,
-			Usage:    "Manifest input file (*.manifest.json)",
-			Action:   cmdargs.EnsureNotEmpty("manifest"),
-		},
-	}...)
+	deployFlags := append(invokeFunctionFlags,
+		cloneFlag(inFlag, true),
+		cloneFlag(manifestFlag, true),
+	)
+	updateFlags := append([]cli.Flag{
+		cloneFlag(inFlag, false),
+		cloneFlag(manifestFlag, false),
+	}, invokeFunctionFlags...)
```

Comment:
BTW, you need to override Action, otherwise it requires both of these flags to be required. Add a test-case where only one of these arguments is provided and check that it works properly.